config/development.json

```diff
@@ -179,6 +179,7 @@
 		"site-indicator": true,
 		"memberships": true,
 		"support-user": true,
+		"titan/free-trial": true,
```

Comment:
Does this need a feature flag at this point?

Reply:
Good point! I was originally hoping to merge this on Thursday or Friday, at which point that made sense.
I've removed the flag and the check.